### PR TITLE
chore(Classroom Monitor): Convert show node info dialog

### LINKED ISSUE #988: Convert show node info dialog to Angular Component
- Create a show node info dialog component
- Everywhere NodeInfoService.showNodeInfo() is called, instead open the new show node info dialog
- Delete NodeInfoService

### PR DESCRIPTION
**Pull request by geoffreykwan**
## Changes

Converted Show Node Info dialog to Angular.

## Test

Make sure the "Step Info" dialog works properly in
- Grade By Step > Click a step > Click "Step Info" in the upper right
- Milestones view > Click a milestone > Click "(Step Info)" link
- Make sure "Open in New Window" button works

Closes #988

### CODE REVIEW

File: src/assets/wise5/classroomMonitor/classroomMonitorComponents/nodeGrading/node-grading-view/node-grading-view.component.ts [hunk 1]
```diff
@@ -377,8 +378,11 @@ export class NodeGradingViewComponent implements OnInit {
     return this.teacherDataService.isWorkgroupShown(workgroup);
   }
 
-  showRubric($event: any) {
-    this.nodeInfoService.showNodeInfo(this.nodeId, $event);
+  showRubric() {
```
Comment (hirokiterashima) [inline src/assets/wise5/classroomMonitor/classroomMonitorComponents/nodeGrading/node-grading-view/node-grading-view.component.ts]:
```suggestion
  protected showRubric(): void {
```

**Review by hirokiterashima — CHANGES_REQUESTED** [review]
Feature works well. 

The header in the step info dialog is slightly different from what's in production.

Production:
Screenshot 2023-01-30 at 9 12 41 AM

This PR:
Screenshot 2023-01-30 at 9 13 54 AM

Can you get it to match? Also, see inline comment.

**Review by breity — APPROVED** [review]
LGTM.

I updated the dialog styles to match the other Angular Material dialogs in CM (student revisions, etc.).

**Review by hirokiterashima — APPROVED** [review]
LGTM

**Pull request merged**